docs/docs/20-quickstart.mdx

```diff
@@ -140,6 +140,24 @@ The Kargo dashboard will be accessible at
 
 The Admin password is `admin`.
 
+:::tip
+```
+Error: INSTALLATION FAILED: failed to authorize: failed to fetch oauth token: unexpected status: 403 Forbidden
+```
+
+If you recieve this error, it's likely because `docker` is configured to
+authenticate with `ghcr.io` using an expired token.
+
+You can confirm this by running:
+```
+% docker pull ghcr.io/akuity/kargo:v0.1.0-rc.21
+Error response from daemon: Head "https://ghcr.io/v2/akuity/kargo/manifests/v0.1.0-rc.21": denied: denied
+```
```

Comment:
All codeblocks have an icon in the upper-right corner for copying their contents...

In light of that, I try to put commands and expected/sample output in two separate boxes so that one doesn't inadvertently copy both the command _and_ the output.

For the same reason, the `%`, which I assume to be a shell prompt placeholder, should also be removed.